### ISSUE TITLE
"Back to Top" button not always identified by Talkbalk

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#606]
Within the “All” tab for “Publications”, the “Back to Top” button that appears when scrolling is not consistently being identified to Talkback. Starting at the top of the “All” view, navigating down will show the “Back to Top” button, but this is not identified to Talkback, and cannot be activated, forcing some users to navigate one publication at a time, all the way back to the top.

Ensure that the “Back to Top” button can be navigated to and activated using a screen reader such as Talkback, otherwise the effort to navigate individually back up through each publication listed to get to the top would be burdensome.

![GitHub19](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/519646ff-890d-409a-8df5-b8a88ad655d6)
